plugins/ban/events/autoban.py

```diff
@@ -0,0 +1,29 @@
+import discord
+from discord.ext import commands
+
+import random
+
+async def execute(
+    ban_plugin,
+    ctx: commands.Context,
+    user: discord.User,
+    reason: str,
+) -> bool:
+    """Execute the autoban event.
+    If the event don't succeds, the function returns False.
```

Comment:
typo

Reply:
Same

Reply:
succeds -> succeeds
Mais je verrai plus "If the event didn't succeed"

Reply:
if the event *doesn't succeeds* (la 3e personne prend un "s", et il manque un "e" au verbe "to succeed")

Reply:
succeed

Reply:
does not* aussi ^^'

Reply:
> if the event *doesn't succeeds* (la 3e personne prend un "s", et il manque un "e" au verbe "to succeed")

Du coup didn't succeed, c'est l'auxiliaire do qui prends la marque du pluriel, pas le participe